### DISCUSSION TITLE
Cannot access Minio via mc after updating to the latest RELEASE.2024-01-31T20-20-33Z

### DISCUSSION THREAD

**Discussion by lukahrobat** [minio/minio#18961] [Q&A] [answered]
Hi,
I have used Minio for a while now and never had issues with access via mc or rclone. I have updated my Minio server to RELEASE.2024-01-31T20-20-33Z and now the access keys don't work anymore. I tried generating new ones but the issue persists

Running command:
`mc ls local`
Returns:
`mc: <ERROR> Unable to list folder. Access Denied.`

Is this related to the security update?

**Comment by harshavardhana** [comment, marked as answer]
Try the workaround for now, we will be making a new release soon.

https://github.com/minio/minio/issues/18939#issuecomment-1921644170
